onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -132,8 +132,10 @@ def check_opset_version(self):
                 .format(opset_version))
             self.model.model.opset_import.remove(ai_onnx_domain[0])
             self.model.model.opset_import.extend([onnx.helper.make_opsetid("", 11)])
+            opset_version = 11
         
         self.fuse_dynamic_quant = True
+        return opset_version
```

Comment:
>opset_version [](start = 15, length = 13)

The function is not supposed to return a value or not?
Line 127 has a simple return.

Reply:
need return the value to make outside logic looks clear.

Reply:
Cool, then how about let 127 also return something to make the function more consistent?

---
In reply to: [489639052](https://github.com/microsoft/onnxruntime/pull/5194#discussion_r489639052) [](ancestors = 489639052)

Reply:
fixed